lib/src/user.dart

```diff
@@ -317,9 +317,9 @@ class FunctionsClient {
   /// Calls a remote function with the supplied arguments.
   /// @name The name of the Atlas function to call.
   /// @functionArgs - Arguments that will be sent to the Atlas function. They have to be json serializable values.
-  Future<dynamic> call(String name, [List<Object?>? functionArgs]) async {
+  Future<dynamic> call(String name, [List<Object?> functionArgs = const []]) async {
```

Comment:
This is technically a breaking change - I'm not opposed to it, but we should at least mention it.

Reply:
Added to change log